jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -18,11 +20,13 @@
                 'tier': 3,
             },
             'logs': {
```

Comment:
I pushed a new commit with those changes. Means we have an `artifact` dict under the treeherder section now, which holds any kind of artifacts to be uploaded and to attach as job info. Beside that there is also a `log_reference` property which references the artifact which is used by TH for parsing the results.

Here the results from a test run:
https://treeherder.allizom.org/logviewer.html#?job_id=2340623&repo=mozilla-central